Makefile

```diff
@@ -1,6 +1,7 @@
-BASENAME=$(shell yq -r '.catalog.name' < dabl-meta.yaml)
-VERSION=$(shell yq -r '.catalog.version' < dabl-meta.yaml)
-SUBDEPLOYMENTS=$(shell yq -r '.subdeployments' < dabl-meta.yaml | sed 's/\[//g' | sed 's/\]//g' | sed 's/,//g')
+BASENAME=$(shell yq -r '.catalog.name' < dabl-meta.yaml 2> /dev/null || yq r dabl-meta.yaml 'catalog.name')
+VERSION=$(shell yq -r '.catalog.version' < dabl-meta.yaml 2> /dev/null || yq r dabl-meta.yaml 'catalog.version')
+SUBDEPLOYMENTS=$(shell yq -r '.subdeployments' < dabl-meta.yaml 2> /dev/null | sed 's/\[//g' | sed 's/\]//g' | sed 's/,//g' \
+	       || yq r dabl-meta.yaml '.subdeployments' | sed 's/\[//g' | sed 's/\]//g' | sed 's/,//g')
```

Comment:
haven't tested this but it appears that you are using a prepended dot (`.`) for `.subdeployments` in both instances whereas you are not doing the same for the `BASENAME` and `VERSION` definitions. May not really matter but just an inconcistency that I spotted